numpy/core/_add_newdocs.py

```diff
@@ -152,6 +152,8 @@
 
 add_newdoc('numpy.core', 'nditer',
     """
+    nditer(op, flags=None, op_flags=None, op_dtypes=None, order='K', casting='safe', op_axes=None, itershape=(), buffersize=0)
```

Comment:
```suggestion
    nditer(op, flags=None, op_flags=None, op_dtypes=None, order='K', casting='same_kind', op_axes=None, itershape=None, buffersize=0)
```
Seems itershape `None` is also supported, so maybe nicer (OTOH its not documented below). I was wondering if we break the long line here, but seems we do not always, so fine to not do it.

So maybe we can just put it in as is, go ahead and merge yourself with or without the suggestion.

Reply:
I prefer showing `()`, since that also gives a clue about the type of the argument. Supporting `None` as an alias of an empty tuple seems weird to me, and I'm wondering how noisy it would be to deprecate it...

Reply:
This spawned #15881 and #15882

Reply:
xref #15885